config/anniversary/sfiii.anniversary.yaml

```
@@ -423,7 +423,7 @@ segments:
       - [0x149C50, asm, sf33rd/Source/Game/end_9]
       - [0x14A1F0, asm, sf33rd/Source/Game/end_main]
       - [0x14B640, c, sf33rd/Source/Game/Entry]
-      - [0x14FB00, asm, sf33rd/Source/Game/Flash_LP] # Good first decomp
+      - [0x14FB00, c, sf33rd/Source/Game/Flash_LP] # Good first decomp
```

Considering you have already decompiled this file the `# Good first decomp` comment should be removed
```suggestion
      - [0x14FB00, c, sf33rd/Source/Game/Flash_LP]
```